Deployment of qiskit.org is failing

Originally added as: "Old event still showing as 'upcoming' on events page"

<img width="1360" alt="Screen Shot 2021-07-20 at 3 48 31 PM" src="https://user-images.githubusercontent.com/79483070/126405207-d7be3a6a-8540-47b1-8121-99f2b7074050.png">

The merge of https://github.com/Qiskit/qiskit-terra/pull/6558 has removed and deployments are failing with:

```
Cloning into 'qiskit-terra'...
cp: cannot stat 'qiskit-terra/qiskit/schemas': No such file or directory
Error: Process completed with exit code 1.
```

--- a/deploy/deploy_cos.sh
+++ b/deploy/deploy_cos.sh
@@ -10,7 +10,7 @@ CONFIG_PATH=$(rclone config file | tail -1)
 # Decrypt config
 openssl aes-256-cbc -K $encrypted_rclone_key -iv $encrypted_rclone_iv -in rclone.conf.enc -out $CONFIG_PATH -d
 
-git clone --depth=1 https://github.com/Qiskit/qiskit-terra.git
-cp -r qiskit-terra/qiskit/schemas ../dist/.
+git clone --depth=1 https://github.com/Qiskit/ibm-quantum-schemas.git
+cp -r ibm-quantum-schemas/schemas ../dist/.
 
 rclone sync --exclude-from ./static-sites.txt ../dist IBMCOS:qiskit-org-web-resources

